webrtc.html

```diff
@@ -2862,6 +2862,18 @@
                     </ol>
                   </li>
                   <li>
+                    <p>Set the <a>[[\ReadyState]]</a> slot of each of
+                    <var>connection</var>'s <code><a>RTCDataChannel</a></code>s
+                    to <code><a data-link-for=
+                    "RTCDataChannelState">"closed"</a></code>.</p>
+                  </li>
+                  <li>
+                    <p>Set the <a>[[\SctpTransportState]]</a> slot of each of
```

Comment:
The [[SctpTransportState]] slot doesn't appear to be defined anywhere.

Reply:
(otherwise this looks good though)

Reply:
Added the definition to PR https://github.com/w3c/webrtc-pc/pull/1620